OutaTime/ViewController.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  ViewController.swift
+//  OutaTime
+//
+//  Created by Nichole Davidson on 1/29/20.
+//  Copyright © 2020 Nichole Davidson. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+    }
+
+
+}
+
```

Comment:
Not using this file. Make sure to delete files you are not using in your project.